### PR TITLE
Added Toasts and some fixes suggested by danish

### PR DESCRIPTION
**Pull request by Ankur9669**

### CODE REVIEW

File: fitbit-react/src/index.js [hunk 1]
```diff
@@ -9,24 +9,27 @@ import { FilterProvider } from "./Context/filter-context";
 import { ProductProvider } from "./Context/products-context";
 import { UserProvider } from "./Context/user-context";
 import { WishListProvider } from "./Context/wishlist-context";
+import { ToastProvider } from "./Context/toast-context";
```
Comment (yaswanthmyneni) [inline fitbit-react/src/index.js:12]: suggestion: create index file and export all contexts from it

File: fitbit-react/src/App.js [hunk 2]
```diff
@@ -8,10 +8,27 @@ import { Route, Routes } from "react-router-dom";
...
+          dispatchToast({
+            type: "ADD_TOAST",
+            payload: {
+              value: { id: uuid(), title: "Test", type: "SUCCESS" },
+            },
+          })
+        }
+      >
+        Add Toasts
+      </button> */}
```
Comment (yaswanthmyneni) [inline fitbit-react/src/App.js]: changes: can remove unused code.

File: fitbit-react/src/Util/Toasts/Toasts.js [hunk 3]
```diff
@@ -13,17 +16,47 @@ function Toasts() {
...
+    //     console.log("Inside", toastList);
+    //     const lastToastElement = toastList[toastList.length - 1];
+    //     deleteToast(lastToastElement.id);
+
+    //     if (toastList.length === 0 && intervalId != "") {
+    //       clearInterval(intervalId);
+    //     }
+    //   }, 100000);
+    // }
+
```
Comment (swstica) [inline fitbit-react/src/Util/Toasts/Toasts.js]: Can remove the commented code

File: fitbit-react/src/Util/Toasts/toasts.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
+.toasts-container{
+    z-index: 1000;
+    position: fixed;
+    top: 1rem;
+    right: 10px;
+}
+.toast{
+    min-width: 200px;
```
Comment (swstica) [inline fitbit-react/src/Util/Toasts/toasts.css]: Can use rem instead of px, especially for 200px.

File: fitbit-react/src/Util/Toasts/Toasts.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+function Toasts() {
+  const { toastList, dispatchToast } = useToast();
+
+  const deleteToast = (id) => {
+    let newToastList = toastList.filter((toast) => toast.id !== id);
+
+    dispatchToast({
+      type: "SET_LIST",
+      payload: newToastList,
```
Comment (yaswanthmyneni) [inline fitbit-react/src/Util/Toasts/Toasts.js:14]: suggestion: better naming like 'SET_TOAST_TO_THE_LIST' helps to understand easily.

**Review by yaswanthmyneni — APPROVED** [review]
you can merge it after changes.

**Review by swstica — APPROVED** [review]
A few minor changes suggested. Rest all looks good to merge.

File: fitbit-react/src/index.js (hunk 1 above)
Comment (Ankur9669) [reply fitbit-react/src/index.js:12]: Ok

File: fitbit-react/src/App.js (hunk 2 above)
Comment (Ankur9669) [reply fitbit-react/src/App.js]: Done

File: fitbit-react/src/Util/Toasts/Toasts.js (hunk 5 above)
Comment (Ankur9669) [reply fitbit-react/src/Util/Toasts/Toasts.js:14]: I feel this name would be very big that is why i kept like that

File: fitbit-react/src/Util/Toasts/Toasts.js (hunk 3 above)
Comment (Ankur9669) [reply fitbit-react/src/Util/Toasts/Toasts.js]: Done

File: fitbit-react/src/Util/Toasts/toasts.css (hunk 4 above)
Comment (Ankur9669) [reply fitbit-react/src/Util/Toasts/toasts.css]: Done

**Pull request merged**